contracts/src/v0.7/Operator.sol

```diff
@@ -186,6 +203,46 @@ contract Operator is AuthorizedReceiver, ConfirmedOwner, LinkTokenReceiver, Oper
     return success;
   }
 
+  /**
+   * @notice Called by the Chainlink node to fulfill requests
+   * @dev Matches functionality of fulfillOracleRequest with the exception of
+   * loading _payment, _callbackAddress, _callbackFunctionId and _expiration from
+   * storage. Ideal for L2's that charge for L1 calldata.
+   * Given params must hash back to the commitment stored from `oracleRequest`.
+   * Will call the callback address' callback function without bubbling up error
+   * checking in a `require` so that the node can get paid.
+   * @param requestId The fulfillment request ID that must match the requester's
+   * @param data The data to return to the consuming contract
+   * @return Status if the external call was successful
+   */
+  function fulfillOracleRequestShort(
+    bytes32 requestId,
+    bytes32 data
+  )
+    external
+    validateAuthorizedSender
+    validateRequestId(requestId)
+    returns (bool)
+  {
+    PackedOracleRequest memory request = s_packedRequests[requestId];
+
+    // WARNING: DO NOT REMOVE THIS CHECK
+    // This check replaces the validateCallbackAddress modifier as callbackAddress is only know after reading storage
+    require(!s_owned[request.callbackAddress], "Cannot call owned contract");
```

Comment:
Summary: Replaced modifier with inline require because callbackAddress is fetched from storage (not known at time of modifier).